services/workflows-service/src/case-management/case-management.service.ts

```diff
@@ -111,6 +111,16 @@ export class CaseManagementService {
             email: ubo.email,
             nationalId: ubo.nationalId,
             percentageOfOwnership: ubo.ownershipPercentage ?? ubo.percentageOfOwnership,
+            role: ubo.role,
```

Comment:
Child workflows have no context schema, hence why validation does not fail on these fields.